README.md

```diff
@@ -10,3 +8,46 @@ To use sbt-npm-dependencies in an existing SBT project with Scala 2.12 or a late
 ```scala
 addSbtPlugin("io.chrisdavenport" %% "sbt-npm-dependencies" % "<version>")
 ```
+
+In your `build.sbt`, enable it with `enablePlugins(NpmDependenciesPlugin)` and declare your dependencies with
+
+```scala
+Compile / npmDependencies ++= Seq(
+  "npmPackageName" -> "version",
+  "secondNpmPackageName" -> "version",
+  ...
+)
+```
+
+## Internals
+
+(For integration with other build systems)
+
+sbt-npm-dependencies will export a file called `NPM_DEPENDENCIES` containing the dependencies in the following format:
+
+```js
+{
+  "compile-dependencies": [
+    { "npmPackageName": "version" },
+    { "secondNpmPackageName": "version" },
+    ...
+  ],
+  "test-dependencies": [
+    { "npmPackageName": "version" },
+    { "secondNpmPackageName": "version" },
+    ...
+  ],
+  "compile-devDependencies": [
+    { "npmPackageName": "version" },
+    { "secondNpmPackageName": "version" },
+    ...
+  ],
+  "test-devDependencies": [
+    { "npmPackageName": "version" },
+    { "secondNpmPackageName": "version" },
+    ...
+  ]
+}
+```
```

Comment:
lol, I think the code is bugged
https://github.com/davenverse/sbt-npm-dependencies/blob/81d0def60664341d9b8a7f9e29a5369eeeacbb28/core/src/main/scala/io/chrisdavenport/sbt/npmdependencies/NpmDependencies.scala#L54-L61